mappings/net/minecraft/network/packet/c2s/common/CookieRequestResponseS2CPacket.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_9091 net/minecraft/network/packet/c2s/common/CookieRequestResponseS2CPacket
```

Comment:
Name should reflect it being C2S

Reply:
Not sure how I ended up doing that, since it renamed and put it in the right package at the same time, either way. This is now fixed,